samples/create_training_pipeline_image_classification_sample.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_create_training_pipeline_image_classification_sample]
+from google.cloud import aiplatform
+from google.protobuf import json_format
+from google.protobuf.struct_pb2 import Value
+
+
+def create_training_pipeline_image_classification_sample(
+    display_name: str, dataset_id: str, model_display_name: str, project: str
+):
+    client_options = dict(api_endpoint="us-central1-aiplatform.googleapis.com")
+    client = aiplatform.PipelineServiceClient(client_options=client_options)
+    location = "us-central1"
+    parent = "projects/{project}/locations/{location}".format(
+        project=project, location=location
+    )
+    training_task_inputs_dict = {
+        "multiLabel": True,
+        "modelType": "CLOUD",
+        "budgetMilliNodeHours": 8000,
+        "disableEarlyStopping": False,
+    }
+    training_task_inputs = json_format.ParseDict(training_task_inputs_dict, Value())
+
+    training_pipeline = {
+        "display_name": display_name,
+        "training_task_definition": "gs://google-cloud-aiplatform/schema/trainingjob/definition/automl_image_classification_1.0.0.yaml",
+        "training_task_inputs": training_task_inputs,
+        "input_data_config": {"dataset_id": dataset_id},
+        "model_to_upload": {"display_name": model_display_name},
+    }
+    response = client.create_training_pipeline(
+        parent=parent, training_pipeline=training_pipeline
+    )
+    print("response")
+    print(" name:", response.name)
```

Comment:
it seems like theres a lot of print statements throughout this sample - is printing all of these things beneficial to the user? If it's a lot of info, would it possibly be more valuable to print the output to some kind of file or to gcs? Y'all know what's best for uCAIP, but I'm just curious about seeing `print` so much here :)

Reply:
+1, if we want to print out the data, maybe use some kind of method like outlined here: https://stackoverflow.com/a/59128615/12375776